docs/user-guide/kubernetes.md

```diff
@@ -859,10 +859,8 @@ Sometimes Træfik runs along other Ingress controller implementations. One such
 The `kubernetes.io/ingress.class` annotation can be attached to any Ingress object in order to control whether Træfik should handle it.
 
 If the annotation is missing, contains an empty value, or the value `traefik`, then the Træfik controller will take responsibility and process the associated Ingress object.
-If the annotation contains any other value (usually the name of a different Ingress controller), Træfik will ignore the object.
 
-It is also possible to set the `ingressClass` option in Træfik to a particular value.
-If that's the case and the value contains a `traefik` prefix, then only those Ingress objects matching the same value will be processed.
+It is also possible to set the `ingressClass` option in Træfik to a particular value. Træfik will only process matching Ingress objects.
```

Comment:
Should add a note that having multiple controllers satisfying an ingress can lead to unintended behavior

Reply:
Added a `!!! note` underneath with a _recommendation_ to prefix ingressClasses.